### DISCUSSION TITLE
[V3] Accessing Component Properties in placeholder method/view

### DISCUSSION THREAD

**Discussion by derekcavaliero** [livewire/livewire#6495] [1. Help]
I have a Livewire v3 component that lazy loads content from a 3rd party REST API (WordPress). 

For example:
`<livewire:article-list lazy per-page="4" wrapper-class="g-col-12 g-col-sm-6 g-col-xxl-3" />`

Inside my component class I have a `placeholder()` method where I load in a separate placeholder view:

`return view('livewire.placeholders.article-list')`

I need to be able to access the `per-page` and `wrapper-class` attribute values so that I can output the proper number of placeholder elements (and with their proper layout classes defined on the wrapper). 

I tried accessing it via `$this` and passing them through to the view not having any luck - I sense I am missing something blatantly obvious but after reading through the documentation I don't see anything saying why I wouldn't be able to access the component attributes/properties inside the placeholder method/view.

**Comment by derekcavaliero** [comment]
Looks like this is a fairly simple fix after diving into the library.

The `SupportLazyLoading` `generatePlaceholderHtml` method needs to simply pass the `$params` to the `placeholder()` method:

```
$placeholder = wrap($this->component)
            ->withFallback($placeholderHtml)
            ->placeholder($params);
```

**Reply by edalzell** [reply]
any chance you could make a PR?

**Reply by derekcavaliero** [reply]
Actively working on it 😄  should have something submitted shortly.

**Reply by derekcavaliero** [reply]
**PR submitted:**
https://github.com/livewire/livewire/pull/6532
